gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/core/logging/utils/LoggingUtils.java

```diff
@@ -35,7 +35,7 @@
 public final class LoggingUtils {
 
     private static final String DEFAULT_EXCLUDED_CONTENT_TYPES =
-        "video.*|audio.*|image.*|application\\/octet-stream|application\\/pdf|text\\/event-stream";
+        "video.*|audio.*|image.*|application/octet-stream|application/pdf|text/event-stream";
```

Comment:
The default excluded types are defined in 2 places. Can it be defined only once?